sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -117,24 +150,42 @@ def get_job_status(self, job_id, **kwargs):
     @distributed_trace
     def cancel_job(self, job_id, **kwargs):
         # type: (str, **Any) -> None
-        """
+        """Cancel a currently processing or queued job.
+
+        A job will not be cancelled if it is already completed, failed, or cancelling.
+        All documents that have completed translation will not be cancelled and will be charged.
+        All pending documents will be cancelled if possible.
```

Comment:
consider changing to 
```suggestion
        If possible, All pending documents will be cancelled.
```